dosomething-dev/fastly-frontend/legacy_redirects_recv.vcl

```diff
@@ -1,14 +1,17 @@
+# Redirect all MX and BR requests to the US site.
+if (req.url ~ "(?i)^\/(mx|br)") { error 800 "/us"; }
+
 # Redirect */actnow/* page requests.
-if (req.url ~ "(?i)^\/((us|mx|br)\/)?actnow\/") { error 800 "/us"; }
+if (req.url ~ "(?i)^\/us\/?actnow\/") { error 800 "/us"; }
 
 # Redirect */programs/* page requests.
-if (req.url ~ "(?i)^\/((us|mx|br)\/)?programs\/") { error 800 "/us"; }
-  
+if (req.url ~ "(?i)^\/us\/?programs\/") { error 800 "/us"; }
+
 # Redirect */project/* page requests.
-if (req.url ~ "(?i)^\/((us|mx|br)\/)?project(s)?\/") { error 800 "/us"; }
+if (req.url ~ "(?i)^\/us\/?project(s)?\/") { error 800 "/us"; }
 
 # Redirect */about/press/* page requests.
-if (req.url ~ "(?i)^\/((us|mx|br)\/)?about\/press\/") { error 800 "/us/about/our-press"; }
+if (req.url ~ "(?i)^\/us\/?about\/press\/") { error 800 "/us/about/our-press"; }
```

Comment:
I think we still need to keep the parenthesis group around the `us/` prefix in all of these so this whole section is marked optional (instead of just the `/`)!

Reply:
Thanks for catching this—restored to dev/QA/prod.